uberfire-rest/uberfire-rest-backend/src/main/java/org/guvnor/rest/backend/ProjectResource.java

```diff
@@ -194,6 +198,7 @@ public Response cloneProject(@PathParam("spaceName") String spaceName,
     @RolesAllowed({REST_ROLE, REST_PROJECT_ROLE})
     public Response createProject(
             @PathParam("spaceName") String spaceName,
+            @HeaderParam("accept-language") Locale locales,
```

Comment:
Wouldn't this constant be accept-language?

https://docs.oracle.com/javaee/7/api/javax/ws/rs/core/HttpHeaders.html#ACCEPT_LANGUAGE